daemon/oci_windows.go

```diff
@@ -105,6 +113,11 @@ func (daemon *Daemon) createSpec(c *container.Container) (*specs.Spec, error) {
 		// which has no WORKDIR and has a COPY file ., . will be interpreted
 		// as c:\. Hence, setting it to default of c:\ makes for consistency.
 		s.Process.Cwd = `C:\`
+
+		// TODO @jhowardmsft LCOW Support.
+		if system.LCOWSupported() && img.OS == "linux" {
```

Comment:
You should have the code that fixes up a spec for LCOW factored out into a separate function and not littered though this one.  In the future its easy to know where to add LCOW spec modifications and lot have to look in multiple places.

Its only a couple options now but when you do runtime it can expand more